api/receiver/client/client.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Add a comment here to explain what this package is for (say something like "client provides an interface to talk to the /api/results/upload endpoint".

Reply:
Done.